developers/blobstream-rollups.md

```diff
@@ -0,0 +1,452 @@
+---
+description: Learn how to build rollups that use Blobstream.
+---
+
+# Introduction to Blobstream rollups
+
+[Blobstream](https://blog.celestia.org/introducing-blobstream/) is the first
+data availability solution for EVM chains that securely scales with the number
+of users. It allows rollups to post their data on Celestia while proving their
+availability in the rollup settlement contract.
+
+This document will outline a few ways to build optimistic or zk-rollups
+that post their data to Celestia and use Blobstream to prove that data's
+availability.
+
+## Concepts
+
+### Share commitment
```

Comment:
Two issues here:
1. It's not clear why "share commitment" is used here instead of "blob commitment" as used everywhere else.
2. For the reader reading from top to bottom, it's not obvious until the next subsection that this is only one way of doing things, rather than the only way.

Reply:
1. I saw it in the specs being called `blob share commitment` that's why I used `share commitment`. Will change ASAP.

2. good catch, will mention that

Reply:
@adlerjohn check here: https://github.com/celestiaorg/docs/pull/1541